Client/Frontend/Browser/ButtonToast.swift

```diff
@@ -107,6 +107,8 @@ class ButtonToast: Toast {
 
             button.snp.makeConstraints { (make) in
                 make.width.equalTo(button.titleLabel!.intrinsicContentSize.width + 2 * ButtonToastUX.ToastButtonPadding)
```

Comment:
Can we try the width multiplier to be 4 instead of 2 to increase the width of the button as well